cicecore/drivers/nuopc/cmeps/ice_prescribed_mod.F90

```diff
@@ -74,13 +56,17 @@ end subroutine ice_prescribed_init
 contains
 !===============================================================================
 
-  subroutine ice_prescribed_init(clock, mesh, rc)
+  subroutine ice_prescribed_init(gcomp, clock, mesh, rc)
 
     ! Prescribed ice initialization
 
-    include 'mpif.h'
+#ifndef SERIAL_REMOVE_MPI
```

Comment:
What is this here for?

Reply:
#ifndef SERIAL_REMOVE_MPI follows the usage around use mpi [elsewhere](https://github.com/search?q=repo%3ACICE-Consortium%2FCICE%20SERIAL_REMOVE_MPI&type=code)

The use mpi is for the mpi_bcast below, which I don't think would be needed if the type were implemented in cicecore/cicedyn/infrastructure/comm/mpi/ice_broadcast.F90 too

Reply:
sorry, I'm still not following. You've added a single broadcast_scalar below (for taxmode). What is it about adding that broadcast that requires this ifdef?

Reply:
There is an mpi_bcast already there a couple lines below the added broadcast_scalar call

Reply:
OK, thanks. I understand.